keel-sqs/src/main/kotlin/com/netflix/spinnaker/config/SqsProperties.kt

```diff
@@ -5,4 +5,6 @@ import org.springframework.boot.context.properties.ConfigurationProperties
 @ConfigurationProperties("sqs")
 class SqsProperties {
   var queueARN: String = ""
+  var visibilityTimeoutSeconds: Int = 60
```

Comment:
Determines how long we have to process the message before it will potentially be visible to another `receiveMessages` call. It looks like the default is zero so we really need to use this to avoid instances reading duplicate messages.

Reply:
In echo I'm using 30 seconds. 60 seems good.